recipes/xorg/all/conanfile.py

```diff
@@ -63,8 +63,6 @@ def system_requirements(self):
                 package_tool.install(update=True, packages=p)
 
     def package_info(self):
-        self.cpp_info.includedirs = []
```

Comment:
why no longer needed?

Reply:
because it's useless (and not allowed) now that we use `self.cpp_info.components[]`. Also, `self.cpp_info.components[name].includedirs = include_dirs` will set an empty directory in one step